### ISSUE TITLE
How to obtain feature maps?

### ISSUE THREAD

**Issue by dhdkc** [sarafridov/K-Planes#4]
Hi, @sarafridov, Thank you for you great work!

In the paper, you mentioned the features of a 4D coordinate are obtained from the k-planes, but I'm curious how you obtain the feature maps of k-planes?

Thank you!

**Comment by Giodiro** [comment]
Hi @dhdkc 
The planes are the trainable parameters of the model (trained with gradient descent)! I'm not sure if you had something more specific in mind?

**Comment by sarafridov** [comment]
Hi @dhdkc, echoing what @Giodiro said and adding that the mathematical description of how the features (stored in the planes) are processed into density and view-dependent color can be found in section 3 of the paper, specifically sections 3.1 and 3.3. Feel free to follow up if this doesn't answer your question!

**Comment by dhdkc** [comment]
> Hi @dhdkc The planes are the trainable parameters of the model (trained with gradient descent)! I'm not sure if you had something more specific in mind?

Thanks so much, now I understood.

**Comment by dhdkc** [comment]
> Hi @dhdkc, echoing what @Giodiro said and adding that the mathematical description of how the features (stored in the planes) are processed into density and view-dependent color can be found in section 3 of the paper, specifically sections 3.1 and 3.3. Feel free to follow up if this doesn't answer your question!

Thanks, that helps a lot!
